documentation/Publications.md

```diff
@@ -0,0 +1,12 @@
+# Publications
+
+The Publications table contains metadata for each publication referred to in the database. 
+Entries must exist in the Publications table prior to inserting data that refer to it.
+The *name* of a publication is required to be unique.
+
+| Column Name | Description  | Unit  | Data Type | Key Type  |
+|---|---|---|---|---|
+| name          | Unique identifier for the publication |   | String(30)  | primary   |
+| bibcode       | Bibcode entry |   | String(100)  |    |
```

Comment:
bibcode or doi should be required

Reply:
I opened issue #98 to address this. Please comment there.